project.clj

```diff
@@ -11,4 +11,5 @@
   :global-vars {*warn-on-reflection* true}
   :dependencies [[org.clojure/tools.reader "1.0.0-beta3"]]
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.9.0-alpha13"]
-                                  [org.clojure/test.check "0.9.0"]]}})
+                                  [org.clojure/test.check "0.9.0"]]}}
+  :monkeypatch-clojure-test false)
```

Comment:
@err this http://clojure.github.io/clojure/branch-master/clojure.spec-api.html#clojure.spec.test/ mentions a runner `clojure.spec.test/run-tests`. I didn't have the cycles to dig deep into it, but I wonder if it might help remove the need for this and simply run generative tests on spec'd functions automatically without us having to build a bunch of deftest wrappers for each one.